### ISSUE TITLE
Compilation warning under Elixir 1.5

### ISSUE THREAD

**Issue by pikeas** [slime-lang/phoenix_slime#49]
```
==> phoenix_slime
Compiling 4 files (.ex)
warning: function Mix.Phoenix.copy_from/5 is undefined or private. Did you mean one of:

      * copy_from/4

Found at 2 locations:
  lib/mix/tasks/phoenix.gen.html.slime.ex:54
  lib/mix/tasks/phoenix.gen.html.slime.ex:60

warning: function Mix.Phoenix.copy_from/5 is undefined or private. Did you mean one of:

      * copy_from/4

  lib/mix/tasks/phoenix.gen.layout.slime.ex:16
```

**Comment by doomspork** [comment]
Howdy @pikeas!  We haven't made any changes to the project yet to support 1.5 or Phoenix 1.3.   If you'd like to get involved any and all help would be appreciated 😁

**Comment by doomspork** [comment]
This was resolved with #50 👍
